cmd/otelcol/config/collector/fargate_config.yaml

```diff
@@ -0,0 +1,159 @@
+            - source_labels: [__name__]
+              regex: '.*grpc_io.*'
+              action: drop
+  signalfx:
+    endpoint: 0.0.0.0:9943
+  zipkin:
+    endpoint: 0.0.0.0:9411
+  hostmetrics:
+    collection_interval: 10s
+    scrapers:
+      cpu:
+      disk:
+      filesystem:
+      memory:
+      network:
+      # System load average metrics https://en.wikipedia.org/wiki/Load_(computing)
+      load:
+      # Paging/Swap space utilization and I/O metrics
+      paging:
+      # Aggregated system process count metrics
+      processes:
+      # System processes metrics, disabled by default
+      # process:
+  smartagent/signalfx-forwarder:
+    type: signalfx-forwarder
+    listenAddress: 0.0.0.0:9080
+  smartagent/ecs-metadata:
+    type: ecs-metadata
+    excludedImages: ${env:IMAGES_TO_EXCLUDE}
+
+processors:
+  batch:
+  # Enabling the memory_limiter is strongly recommended for every pipeline.
+  # Configuration is based on the amount of memory allocated to the collector.
+  # In general, the ballast should be set to 1/3 of the collector's memory, the limit
+  # should be 90% of the collector's memory up to 2GB. The simplest way to specify the
+  # ballast size is set the value of SPLUNK_BALLAST_SIZE_MIB env variable. Alternatively, the
+  # --mem-ballast-size-mib command line flag can be passed and take priority.
+  # For more information about memory limiter, see
+  # https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiter/README.md
+  memory_limiter:
+    check_interval: 2s
+    limit_mib: ${SPLUNK_MEMORY_LIMIT_MIB}
+  # detect if the collector is running on a cloud system
+  # important for creating unique cloud provider dimensions
+  resourcedetection:
+    detectors: [ecs]
+    override: false
+  # Same as above but overrides resource attributes set by receivers
+  resourcedetection/internal:
+    detectors: [ecs]
+    override: true
+  # Enables the filter processor with example settings
+  # Full configuration here: https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/filterprocessor
+  # NOTE: These settings need to be change when using this processor
+  filter/1:
+    metrics:
+      exclude:
+        match_type: regexp
+        metric_names: ${env:METRICS_TO_EXCLUDE}
+  # Optional: The following processor can be used to add a default "deployment.environment" attribute to the logs and
+  # traces when it's not populated by instrumentation libraries.
+  # If enabled, make sure to enable this processor in the pipeline below.
+  #resource/add_environment:
+    #attributes:
+      #- action: insert
+        #value: staging/production/...
+        #key: deployment.environment
+
+exporters:
+  # Traces
+  sapm:
+    access_token: "${SPLUNK_ACCESS_TOKEN}"
+    endpoint: "https://ingest.${SPLUNK_REALM}.signalfx.com/v2/trace"
+  # Metrics + Events
+  signalfx:
+    access_token: "${SPLUNK_ACCESS_TOKEN}"
+    realm: "${SPLUNK_REALM}"
+    sync_host_metadata: true
```

Comment:
this option requires  `resourcedetection` -> `detectors: system`. If it's possible/valuable to fetch host metrics in fargate, we should add system detector. Otherwise, this line should be removed.

Reply:
I removed the line.